templates/web/bexley/waste/summary_request.html

```diff
@@ -0,0 +1,55 @@
+[%
+  title = 'Submit container request';
+  thing = 'container request';
+  summary_title = 'Container requests';
+  step1 = 'request';
+
+  data = form.saved_data;
+  requests = c.cobrand.requests_for_display(data);
+%]
+
+[% BLOCK answers %]
+  [% IF data.household_size %]
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key govuk-summary-list__key--sub">Household size</dt>
+      <dd class="govuk-summary-list__value">[% data.household_size %] person(s)</dd>
+    </div>
+  [% END %]
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key govuk-summary-list__key--sub">Reason</dt>
+    <dd class="govuk-summary-list__value">[% data.request_reason %]</dd>
+  </div>
+
+  [% # Deliveries %]
+  [% IF requests.0.size %]
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key govuk-summary-list__key--sub"><b>Deliver new or replacement:</b></dt>
+    </div>
+    [% FOR container IN requests.0 %]
+      [% INCLUDE container_row container = container %]
```

Comment:
You don't need  the `container = container`

Reply:
Yeah, this was probably me trying to make things work before I realised that passing-by-reference was what was causing me issues.